resources/views/layouts/app.blade.php

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
-<html lang="vi">
+<html lang="en">
```

Comment:
```suggestion
<html lang="{{ app()->getLocale() }}">
```
Nếu mình đổi ngôn ngữ thì sao?